app/api/user/service.py

```diff
@@ -10,3 +11,17 @@ def get_user_by_id(db: Session, user_id: UUID):
 def delete_user_by_id(db: Session, user_id: UUID):
     db.query(models.User).filter(models.User.id == user_id).delete()
     db.commit()
+
+
+def update_user_by_id(db: Session, user_id: UUID, user_update: schemas.User):
+    user_info = db.query(models.User).filter(models.User.id == user_id).first()
+    if user_update.email:
+        user_info.email = user_update.email
+    if user_update.username:
+        user_info.username = user_update.username
+    if user_update.address:
+        user_info.address = user_update.address
+    if user_update.cellphone_number:
+        user_info.cellphone_number = user_update.cellphone_number
+    db.commit()
+    return db.query(models.User).filter(models.User.id == user_id).first()
```

Comment:
```suggestion
    db.query(User) \
      .filter(id=user_id) \
      .update(payload.dict(), synchronize_session="fetch")

    updated_user = get_user_by_id(db, user_id)
    return updated_user
```

`sqlalchemy` 的 `Query.update()` 可以傳字典進去，一次改變多個欄位的值

請參考：
- https://docs.sqlalchemy.org/en/14/orm/session_basics.html#update-and-delete-with-arbitrary-where-clause
- https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.update